### ISSUE TITLE
[Batch] Portfolio

### ISSUE THREAD

**Issue by shakingbeef** [rubyforgood/stocks-in-the-future#118]

**Comment by ramirez-gabriela27** [comment]
Tables for Stocks and Portfolios have been created and I'm refining the models we have created for them.

I am currently working on functionality to add and remove stocks from a portfolio, once that is done, I can integrate the work I have on a draft branch to pull the stock info form the [Alpha Vantage API](https://www.alphavantage.co/documentation/)

My goal is to have this done, as well as a Portfolio UI by the time we meet on Thursday.

**Comment by h-m-m** [comment]
Closing in favor of https://github.com/rubyforgood/stocks-in-the-future/issues/146
